app/src/main/java/uk/gov/onelogin/login/UriBuilder.kt

```diff
@@ -12,7 +12,7 @@ data class UriBuilder(
 ) {
   val url: Uri = Uri.parse(baseUri)
             .buildUpon().appendQueryParameter("response_type", "code")
-            .appendQueryParameter("scope", "openid email phone offline_access")
+            .appendQueryParameter("scope", "openid offline_access")
```

Comment:
My bad - but the ticket was incorrect - the scope should just be `openid`

We corrected this when we did the integration between iOS and the stub but I forgot to go back and update the Android tickets. 

That's done now, please can we update this accordingly? Thank you and sorry for catching this so late!

Reply:
<img width="527" alt="Screenshot 2023-11-08 at 16 14 21" src="https://github.com/govuk-one-login/mobile-android-one-login-app/assets/39162664/012e7536-ea38-41c1-b101-043d6d014614">

Reply:
no problem , will update that now